modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -785,5 +785,46 @@ definition {
 			PRMMDFRequest.assertClaimMessage(key_claimMessage = "You can submit up to 2 claim(s).");
 		}
 	}
+	@description = "This is a test for LPS-178087. Verify that it is possible to edit a created MDF"
+	@priority = 5
+	test CanEditMDF {
+		task ("Given I go to add a new MDF Request") {
+		    JSONPRM.createMDFRequest();
+		}
+
+		task ("When the manager user approves the submitted MDF") {
```

Comment:
Hi @MGabriellaS, here you are saying that the manager approves the MDF, but below you don't do that. So you can remove this from your task!